Source/WebCore/editing/markup.cpp

```diff
@@ -1170,9 +1170,8 @@ Ref<DocumentFragment> createFragmentFromText(const SimpleRange& context, const S
     if (text.isEmpty())
         return fragment;
 
-    String string = text;
-    string.replace("\r\n", "\n");
-    string.replace('\r', '\n');
+    String string = makeStringByReplacingAll(text, "\r\n"_s, "\n"_s);
+    string = makeStringByReplacingAll(string, '\r', '\n');
```

Comment:
This idiom happens in 3 places. I think we may want to find a place to put a shared function that does this.